PennyMe/ArtworkViews.swift

```diff
@@ -45,16 +49,16 @@ class ArtworkMarkerView: MKMarkerAnnotationView {
                     displayPriority = MKFeatureDisplayPriority.required
                 }
                 
-                // Set marker color
-                markerTintColor = artwork.markerTintColor
-                
-                // Create view when marker is pressed
-                let identifier = "marker"
+                // define annotation view
                 var view: MKMarkerAnnotationView
+                let identifier = artwork.title
```

Comment:
this might have solved the issue, but I'm not sure (previously the identifier was just "marker" for all of them)